airsonic-main/src/main/java/org/airsonic/player/dao/UserDao.java

```diff
@@ -71,7 +72,7 @@
     private final String userTableQuote;
 
     @Autowired
-    public UserDao(String userTableQuote) {
+    public UserDao(@Value("${DatabaseUsertableQuote:}") String userTableQuote) {
```

Comment:
Hmmm, I disagree, what property is being duplicated again?

The notion of creating a "bean" for a String is just weird. Which then ends up being supplied to the actual constructors that need it again. So instead of 2 places, now that String is in 3 places,.

Let's say I create a "bean", this constructor still needs something for that bean to be injected here, right? So you still end up putting an annotation here anyway and in the other constructor. And then on top of that you now have a managed bean, that is nothing but a String from the environment. What's being duplicated more again?